index.html

```diff
@@ -10,33 +10,38 @@
   <header>
 	<label><a href="#" class="logo">My Logo</a></label>
 	  <nav>
-	    <ul>
-		  <li><a href="#" class="bar">
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+	    <div class="bar">
 		    <div class="b1"></div>
 			<div class="b2"></div>
-			<div class="b3"></div></a>
-		  </li>
-		</ul>
+			<div class="b3"></div>
+		</div>
 	  </nav>	
   </header>
-	<main class="backimg">
-	  <section class="h">
+	<main>
+	  <section>
+      <div class="container">
 		 <h1>I’m Mariane<br>Glad to see<br>you!</h1>
 		   <p class="p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 			 <p class="p2">LET’S CONNECT</p>
-	        <ul class="images">
-			    <li><a href=""><img src="img/Vector.png" alt="icon1"></a></li>
-			    <li><a href=""><img src="img/Linkedin.png" alt="icon2"></a></li>
-			    <li><a href=""><img src="img/Vector(1).png" alt="icon3"></a></li>
-			    <li><a href=""><img src="img/Vector(2).png" alt="icon4"></a></li>
-			    <li><a href=""><img src="img/Vector(3).png" alt="icon5"></a></li>
+	        <ul class="icons">
+			    <li><a href=""><img src="img/Vector.png" alt="icon1" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Linkedin.png" alt="icon2" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Vector(1).png" alt="icon3" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Vector(2).png" alt="icon4" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Vector(3).png" alt="icon5" class="icon-item"></a></li>
 	       </ul>
+         </div>
 	    </section>
     </main>
```

Comment:
- [ ] From 768px your design looks a bit different, kindly add the right media query and align content to look like the Figma design


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 1-Main (1)](https://user-images.githubusercontent.com/69638013/165751428-63cf1170-9f23-4783-8c5e-60dc18f1e7d0.png) | ![Screenshot_91](https://user-images.githubusercontent.com/69638013/165751328-53ff9393-bf1e-461f-935b-96b9094abefd.png)